cypress/e2e/testBoards.cy.js

```diff
@@ -55,6 +55,7 @@ describe("Boards", function() {
 		it("Delete a Board", function() {
 			cy.visit('/u/'+this.credentials.userName +'/boards');
             cy.boardDelete(this.credentials.boardName);
+            cy.wait(1500);
```

Comment:
In the future try to wait for the appearance/display of an UI element or the response of a API request.